Apromore-Custom-Plugins/CSVImporter-Logic/src/main/java/org/apromore/service/csvimporter/services/ParquetFactoryProvider.java

```diff
@@ -36,7 +36,8 @@ public ParquetImporterFactory getParquetFactory(String fileExtension) {
         LOGGER.info("File Format: " + fileExtension);
         if (fileExtension.equalsIgnoreCase(CSV_FILE_EXTENSION)) {
```

Comment:
Can we reverse the if conditions. 
I know the line is added , its a small work to reverse the if condition

Reply:
Also can be moved to switch case

Reply:
Rewrote as a switch.